modules/rollback-operator-upgrades.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-operator.adoc
+:_content-type: PROCEDURE
+[id="rollback-operator-upgrades_{context}"]
+= Rolling back an Operator upgrade
+
+When rolling back to a previous Operator version, you must perform some manual steps because of the way that {olm-first} is configured to perform updates for {product-title-short}.
+You can roll back an {product-title-short} Operator to a previous version; for example, if you experience problems after upgrading.
+
+.Procedure
+
+. Delete the {olm} subscription by running the following command:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete subscription rhacs-operator
+----
+. Delete the cluster service version (CSV) by running the following command:
++
+[source,terminal]
+----
+$ kubectl -n rhacs-operator delete csv -l operators.coreos.com/rhacs-operator.rhacs-operator
+----
+. Determine the previous version you want to roll back to by choosing one of the following options:
+* If the current Central instance is running, query the {product-title-short} API to get the rollback version by running the following command:
++
+[source,terminal]
+----
+$ curl -k -s -u <user>:<pass> https://<central hostname>/v1/centralhealth/upgradestatus | jq -r .upgradeStatus.forceRollbackTo
+----
+* If the current Central instance is not running, perform the following steps:
+.. Ensure the Central deployment is scaled down by running the following command:
++
+[source,terminal]
+----
+$ kubectl scale -n <central namespace> –replicas=0 deploy/central
+----
+.. Create a pod in your Central namespace using the following pod spec:
++
+[source,yaml]
+----
```

Comment:
I tested out this yaml, and there are a few issues with it.  Can you replace it with the following yaml:

```kind: Pod
apiVersion: v1
metadata:
  name: get-previous-db-version
spec:
  containers:
  - name: get-previous-db-version
    image: registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:<rollback version>
    command:
    - sh
    args:
    - '-c'
    - "cat /var/lib/stackrox/.previous/migration_version.yaml | grep '^image:' | cut -f 2 -d : | tr -d ' '"
    volumeMounts:
    - name: stackrox-db
      mountPath: /var/lib/stackrox
  volumes:
  - name: stackrox-db
    persistentVolumeClaim:
      claimName: stackrox-db
```

Reply:
This is giving me an error: "An error occurred - No "kind" field found in YAML."

Reply:
Thanks for catching that -- copy/paste error!